xml/ay_bigfile.xml

```diff
@@ -17165,44 +17161,65 @@ mv /var/run/zypp.sav /var/run/zypp.pid
   <title>Advanced Linuxrc Options</title>
   <info/>
   <para>
-   Linuxrc is a program used for setting up the kernel for installation
-   purposes. It allows the user to load modules, start an installed system,
-   a rescue system or an installation via &yast;.
+   <command>Linuxrc</command> is a small program that runs after the kernel loads, but before &ay;
```

Comment:
spelling of 'linuxrc')? see comment above...

In this case, I would prefer present perfect: "... that runs after the kernel has loaded..."